aggregate/aggregation_helpers.py

```diff
@@ -35,10 +35,10 @@ def order_aggregated_columns(
         if not household:
             for ind_category in categories[ind]:
                 for race_crosstab in categories["race"]:
-                    for measure in ["", "_pct"]:
+                    for measure in ["_count", "_pct"]:
                         column_label_base = f"{ind_category}_{race_crosstab}{measure}"
```

Comment:
this needs a refactor, race should come after measure and variance. I can make this fix as I know this function best

Reply:
my apologies I don't think that feedback is correct, I got mixed up on where this token goes